.changeset/olive-foxes-own.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/astro": patch
+---
+
+Introduce `TransformStream` and work directly with encoded chunks when modifiying stream
```

Comment:
Generally we prefer to write more "product" oriented changesets when possilble. In this case I'd mentioned the performance boost, instead of mentioning the "how" with `TransformStream`

Reply:
I'll keep that in mind, thanks! and updated